controller/work_item_link_type.go

```diff
@@ -84,106 +58,6 @@ func enrichLinkTypeList(ctx *workItemLinkContext, list *app.WorkItemLinkTypeList
 			Related: &relatedURL,
 		}
 	}
-	// Build our "set" of distinct category IDs already converted as strings
```

Comment:
I'm removing them because UI never made use of it. Also, the space inclusion is hard to maintain because sooner or later a link type is no longer tied to a space but a space template. For now the only space we've included is the system space anyways, so there was no real use for that.